### DISCUSSION TITLE
Regarding chat option

### DISCUSSION THREAD

**Discussion by amoljagdalepucsd** [Ylianst/MeshCentral#5126] [Q&A]
Hi community ,
Is it possible to open chat browser window in bottom right corner on remote system.
I am trying to change code in meshcore.js inside openUserDesktopUrl function while executing task object but not getting any break.
                var task = { name: 'MeshChatTask', user: user, domain: domain, execPath: process.env['windir'] + '\\system32\\cmd.exe', arguments: ['/C START ' + url.split('&').join('^&')] };

**Comment by amoljagdalepucsd** [comment]
@Ylianst @krayon007 @si458 
do you have any thoghts ?

**Reply by si458** [reply]
im not sure how the chat system works sorry, ive never every used it in 2 years

**Reply by amoljagdalepucsd** [reply]
okay @si458  Thanks for your response
